automatic/scribus/tools/chocolateyInstall.ps1

```diff
@@ -1,9 +1,17 @@
-﻿$packageName = 'scribus'
-$fileType = 'exe'
-$silentArgs = '/S'
-# {\{DownloadUrlx64}\} gets “misused” here as 32- and 64-bit link array due to limitations of Ketarin/chocopkgup
-$urlArray = {{DownloadUrlx64}}
-$url = $urlArray[0]
-$url64bit = $urlArray[1]
-
-Install-ChocolateyPackage $packageName $fileType $silentArgs $url $url64bit
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  packageName    = 'scribus'
+  fileType       = 'exe'
+  file           = "$toolsPath\scribus-1.4.6-windows.exe"
+  file64         = "$toolsPath\scribus-1.4.6-windows-x64.exe"
+  softwareName   = ''
```

Comment:
ops, yeah something is supposed to go there.
I must have just forgotten about it.
That's what I get for working on multiple packages at the same time. XD